db/models.py

```diff
@@ -1,4 +1,7 @@
+from django.contrib.auth.models import AbstractUser
```

Comment:
Code Style Issue: You are directly importing `AbstractUser` from `django.contrib.auth.models`. It's a better practice to use `get_user_model()` function from `django.contrib.auth` to get the user model. This function will always return the currently active user model – the custom user model if one is specified, or `User` otherwise.

Reply:
But I need id to inherit from it for my User model

Reply:
Requirement was to user AbstractUser to inherit from